<launch>
	<arg name="openni2_launch" default="false" />
	<!-- true if already launched -->
	<arg name="camera_id" default="kinect" />
	<arg name="is_srv_used" default="false" />
	<!-- start the tracking with an service call -->

	<arg name="tf_plane" default="false" />
	<!-- tf_plane use only with robot, odom to camera_frame must be available -->

	<arg name="publish_wuser_skeleton" default="true" />
	<arg name="track_wave_user" default="true" />
	<!-- true if you want to track a waving user -->

	<arg name="pub_filtered_pc" default="false" />
	<!-- is depricated keep it here just in case -->

	<include file="$(find openni2_launch)/launch/openni2.launch" unless= "$(arg openni2_launch)">
		<arg name="camera" value="$(arg camera_id)" />
	</include>
	<node pkg="nodelet" type="nodelet" name="squirrel_person_tracker"
		args="load squirrel_person_tracker/squirrel_person_tracker_nodelet /$(arg camera_id)/$(arg camera_id)_nodelet_manager"
		output="screen" cwd="ROS_HOME">
		<param name="camera_optical_frame_id" type="str"
			value="$(arg camera_id)_depth_optical_frame" />
		<param name="is_srv_used" type="bool" value="$(arg is_srv_used)" />
		<param name="tf_plane" type="bool" value="$(arg tf_plane)" />
		<param name="pub_filtered_pc" type="bool" value="$(arg pub_filtered_pc)" />
		<param name="publish_wuser_skeleton" type="bool"
			value="$(arg publish_wuser_skeleton)" />
		<param name="track_wave_user" type="bool" value="$(arg track_wave_user)" />
	</node>

	<node pkg="spt_helper" type="spt_user_info" name="matchstick_man" />

</launch>
